js/ui/drop_down_button.js

```diff
@@ -518,9 +518,22 @@ const DropDownButton = Widget.inherit({
         }
     },
 
+    _getItemByKey() {
+        const key = this._getKey();
+        const selectedItemKey = this.option('selectedItemKey');
+        const items = this.option('items');
+        const filter = item => (key === 'this' ? item : item[key]) === selectedItemKey;
+
+        return items?.find(filter) || null;
```

Comment:
IE11 [does not support](https://caniuse.com/#feat=array-find) it :(

Reply:
fixed